Source/WebInspectorUI/UserInterface/Controllers/NetworkManager.js

```diff
@@ -328,6 +337,11 @@ WI.NetworkManager = class NetworkManager extends WI.Object
         loadAndParseSourceMap();
     }
 
+    isDownloadingSourceMapURL(url)
```

Comment:
This method is accessing URLs in two different buckets, despite its name. 
Since the intent is to know whether the URL is for a source map url, regardless of its state, how about renaming to `isSourceMapURL(url)`?

Reply:
I did change to `isSourceMapURL(url)`, but there is a `Map` on the `networkManager` called `_sourceMapURLMap` that has all of the identified source map URLs. This raises a question of whether it is better to reference that instead, or continue to only look at URLs where a download attempt has occurred, or a download failed. The correct answer still seems like the latter, but I think either approach will have the same behavior result in practice.